src/fights.ts

```diff
@@ -2177,7 +2177,10 @@ export function printEmbezzlerLog(): void {
 
 const isFree = (monster: Monster) => monster.attributes.includes("FREE");
 const valueDrops = (monster: Monster) =>
-  sumNumbers(itemDropsArray(monster).map(({ drop, rate }) => garboValue(drop) * rate));
+  sumNumbers(itemDropsArray(monster).map(({ drop, rate }) => garboValue(drop) * rate)) +
+  (have($familiar`Robortender`)
+    ? Robortender.dropChance() * garboValue(Robortender.dropFrom(monster))
```

Comment:
you were saying that when you tested it didn't fight a penguin. two followups for you via this code:

- we've fully validated that dropChance & garbo value are playing nicely in multiplication, right? could see one being an int and the other being a float and some stupid bullshit happening with rounding if it's not a 100% chance maybe.
- do you have a black crayon penguin in your locket? i assume yes. i just realized i actually don't, so that is on my list of things to do today too lol

might be worth printing this for testing when it tabulates it just so you can have a reference for what garbo thinks it's worth just in case there's a valuations issue here...

Reply:
js doesn't have ints. I'll test it again on casual leg

Reply:
thank you for resolving this. i am proud of myself that i had identified something that was adjacent to the actual problem even though it extremely was not the actual problem. i needed a win today.